### DISCUSSION TITLE
Scheda Supervisor

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1391] [HUB: Home Assistant]
Ciao ho appena installato ha su Raspberry4, in modalità OS (HassOS/HASSIO). sono nelle prime fasi e volevo installare gli addon essenziali come consigliato dalla guida, ma non trovo la scheda supervisor, devo impostare quaclosa per vederla? grazie

**Comment by indomus** [comment]
Impostazioni > Componenti aggiuntivi, backup e supervisor (leggi bene nelle descrizioni delle varie righe) 
Abbiamo anche aggiornato [la guida](https://indomus.it/formazione/gli-imprescindibili-gli-add-on-essenziali-da-installare-su-hassio/), ricaricala.

**Reply by mriitan** [reply]
perfetto, trovata, davvero da scemo.
Altro problemino, HACS lo devo integrare attraverso file zip da scaricare nella directory del rasp, ma come accedo al rasp? posso farlo dal pc di appoggio, via ftp? con un mac sempre via ftp? le credenziali? sono nel panico, un pò di clemenza

**Reply by mriitan** [reply]
ho capito lo faccio dal editor che ho appena scaricato, giusto?
